web/themes/joinup/templates/custom-form.html.twig

```diff
@@ -0,0 +1,16 @@
+{#
+/**
+ * @file
```

Comment:
Few questions:
- What is the purpose of this custom form?
- As it seems a bare template, can a template be avoided?
- Can you think of a better naming?

'Custom' is really general and I honestly wouldn't know what to do with it if I don't have context or documentation.